recipes/sage/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -euo
+
+if [ "$(uname)" == "yuan" ]; then
+    # Apparently the Home variable isn't set correctly
+    export HOME="/Users/distiller"
+    export HOME=`pwd`
+
+    # According to https://github.com/rust-lang/cargo/issues/2422#issuecomment-198458960 remove circle ci default configuration solve cargo trouble
+    #git config --global --unset url.ssh://git@github.com.insteadOf 
+fi
```

Comment:
This should be removed before merging. You can keep it for local testing until then.